pkg/network/multus.go

```diff
@@ -36,6 +43,51 @@ func changeSafeMultus(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
 	return nil
 }
 
+// cleanUpMultus checks specific multus outdated objects or ones that are no longer compatible and deletes them.
+func cleanUpMultus(conf *opv1alpha1.NetworkAddonsConfigSpec, ctx context.Context, client k8sclient.Client) []error {
+	if conf.Multus == nil {
+		return nil
+	}
+
+	errs := []error{}
+	// deletes the existing multus object by specific name
+	errs = append(errs, cleanUpMultusOldName(ctx, client)...)
+
+	return errs
+
+}
+
+// cleanUpMultusOldName deletes multus ds object with old name after a new name was introduces in version 0.25.0.
+// TODO This function is required as long as we support upgrade from version <= 0.25.0.
```

Comment:
Please drop the TODO and keep only the rest based on Dan's comments

Reply:
changing according to dan's comment to REQUIRED-FOR ( example in https://github.com/oVirt/vdsm/blob/f6a1ba0a0e232a28db6c38a3cdecf1a5cc96105a/lib/vdsm/network/kernelconfig.py#L232)